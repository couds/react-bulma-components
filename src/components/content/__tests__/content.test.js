import React from 'react';
import { render } from 'react-testing-library';
import { Content } from '..';

describe('Content component', () => {
  it('should render', () => {
    const { asFragment } = render(<Content />);
    expect(asFragment()).toMatchSnapshot();
  });
});
