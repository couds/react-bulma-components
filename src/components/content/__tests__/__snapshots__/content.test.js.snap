// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content component Should have content classname 1`] = `
<div
  className="content"
  style={Object {}}
>
  <p
    className="bd-notification is-success"
  >
    <p>
      Default
    </p>
    <p>
      Container
    </p>
  </p>
</div>
`;
